master/docs/manual/secretsmanagement.rst

```diff
@@ -59,11 +59,13 @@ SecretInAFile
 
 .. code-block:: python
 
-    c['secretsProviders'] = [util.SecretInAFile(dirname="/path/toSecretsFiles"]
+    c['secretsProviders'] = [util.SecretInAFile(dirname="/path/toSecretsFiles",
+                                                strip=True)]
 
 In the passed directory, every file contains a secret identified by the filename.
 
-e.g: a file ``user`` contains the text ``pa$$w0rd``.
+e.g: a file ``user`` contains the text ``pa$$w0rd``. Trailing newlines are
```

Comment:
please look at other part of the doc for example on how we describe the signature of a step ex:
```

.. py:class:: buildbot.steps.source.git.Git

The :bb:step:`Git` build step clones or updates a `Git <http://git.or.cz/>`_ repository and checks out the specified branch or revision.


.. note::

    The Buildbot supports Git version 1.2.0 and later: earlier versions (such as the one shipped in Ubuntu 'Dapper') do not support the :command:`git init` command that the Buildbot uses.

::

    from buildbot.plugins import steps

    factory.addStep(steps.Git(repourl='git://path/to/repo', mode='full',
                              method='clobber', submodules=True))

The Git step takes the following arguments:

``repourl``
   (required): the URL of the upstream Git repository.

``branch``
   (optional): this specifies the name of the branch to use when a Build does not provide one of its own.
   If this parameter is not specified, and the Build does not provide a branch, the default branch of the remote repository will be used.

[...]
```